sig-architecture/annual-report-2021.md

```diff
@@ -87,51 +147,84 @@ In future, this will be generated from delta of sigs.yaml from $YYYY-01-01 to $Y
+- [Production Readiness](https://github.com/kubernetes/community/tree/master/sig-architecture#production-readiness-1)
+    - Production readiness review [became mandatory] for enhancements in 2021, starting with the 1.21 release cycle.
+    - We added a fourth approver, Elana Hashman, who joined the team after shadowing in 1.21.
+    - We created documentation on the [lifecycle of feature gates].
+    - We reviewed data from the 2020 Production Readiness Survey, and prepared a 2021 survey, whose [responses we analyzed]. # **TODO:** @dims to update link to point to YouTube recording
+    - The team implemented a [soft deadline] to set expectations around what can be reviewed in time for enhancements freeze.
+    - We reviewed a total of 171 KEPs in 2021, averaging 16 per approver per release:
+        - 51 KEPs in 1.21 (3 approvers)
+        - 52 KEPs in 1.22 (4 approvers)
+        - 68 KEPs in 1.23 (4 approvers)
+
+[became mandatory]: https://groups.google.com/g/kubernetes-dev/c/GvT-qOaVsWE/m/zeazt2kODQAJ
+[soft deadline]: https://groups.google.com/g/kubernetes-dev/c/EvpXsyLf5E0/m/D4Jhc7ItAgAJ
+[responses we analyzed]: https://docs.google.com/document/d/1BlmHq5uPyBUDlppYqAAzslVbAO8hilgjqZUTaNXUhKM/edit#bookmark=id.g9stoq3xr2dx
+[lifecycle of feature gates]: https://git.k8s.io/community/contributors/devel/sig-architecture/feature-gates.md
+     
 
-New in $YYYY:
-- [$wg-name](https://git.k8s.io/community/$wg-id/) ([$YYYY report](https://git.k8s.io/community/$wg-id/annual-report-$YYYY.md))
--
+## Working groups
 
-Retired in $YYYY:
-- [$wg-name](https://git.k8s.io/community/$wg-id/) ([$YYYY report](https://git.k8s.io/community/$wg-id/annual-report-$YYYY.md))
--
+- [API Expression](https://git.k8s.io/community/wg-api-expression/) ([2021 report](https://git.k8s.io/community/wg-api-expression/annual-report-2021.md))
 
-Continuing:
-- [$wg-name](https://git.k8s.io/community/$wg-id/) ([$YYYY report](https://git.k8s.io/community/$wg-id/annual-report-$YYYY.md))
--
+TODO: dims to send email to WG asking them to talk to us sync of async
 
 ## Operational
 
 Operational tasks in [sig-governance.md]:
 
-- [ ] [README.md] reviewed for accuracy and updated if needed
-- [ ] [CONTRIBUTING.md] reviewed for accuracy and updated if needed
+- [x] [README.md](https://github.com/kubernetes/community/blob/master/sig-architecture/README.md) reviewed for accuracy and updated if needed
+- [ ] [CONTRIBUTING.md](https://hackmd.io/GY081dMNThS16WWWB46unw) reviewed for accuracy and updated if needed
       (or created if missing and your contributor steps and experience are different or more
       in-depth than the documentation listed in the general [contributor guide] and [devel] folder.)
-- [ ] Subprojects list and linked OWNERS files in [sigs.yaml] reviewed for accuracy and updated if needed
+	  **We need to create this:**
+	- [ ] [HackMD](https://hackmd.io/GY081dMNThS16WWWB46unw) for template started - Riaan
```

Comment:
Is this a TODO?

Reply:
Yes, will work on it this week

Reply:
@sftim @dims 
I update the [CONTRIBUTING.md](https://hackmd.io/GY081dMNThS16WWWB46unw) as a draft.
Please preview and propose changes as you see fit, then I will update stick it in a PR for review.

Reply:
@Riaankl the new hackmd you pointed to, looks good.

Reply:
I will make a PR for it, then we can add a real link to the anual report.

Reply:
thanks @Riaankl

Reply:
linking to the merged Contributing.md